api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -685,15 +697,33 @@ private boolean userHasEverLoggedIn(
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
-  private void updateProfileForUser(DbUser user, Profile updatedProfile, Profile previousProfile) {
+  public ResponseEntity<Void> adminUpdateProfile(DbUser user, Profile updatedProfile) {
+    // Save current profile for audit trail.
+    final Profile previousProfile = profileService.getProfile(user);
+
     validateUpdatedProfile(updatedProfile, previousProfile);
 
-    if (!userProvider.get().getGivenName().equalsIgnoreCase(updatedProfile.getGivenName())
-        || !userProvider.get().getFamilyName().equalsIgnoreCase(updatedProfile.getFamilyName())) {
-      userService.setDataUseAgreementNameOutOfDate(
-          updatedProfile.getGivenName(), updatedProfile.getFamilyName());
+    profileService.adminUpdateProfile(updatedProfile);
+
+    // update the user's free-tier-active status, if this changes
+    if (!updatedProfile.getFreeTierDollarQuota().equals(previousProfile.getFreeTierDollarQuota())) {
+      freeTierBillingService.setFreeTierDollarOverride(
+          user, updatedProfile.getFreeTierDollarQuota());
     }
 
+    if (workbenchConfigProvider.get().featureFlags.requireInstitutionalVerification) {
```

Comment:
users can't update their own affiliation

Reply:
So is the idea that a profile update never touches the verified inst affl? If that's true, then I don't understand why it's part of this controller method.